bool start_check
---
float32[] positionsX
float32[] positionsY

